docs/en-US/component/time-select.md

```diff
@@ -29,22 +49,23 @@ time-select/time-range
+| editable                              | whether the input is editable                            | boolean            | —                                                                                      | true        |
+| clearable                             | whether to show clear button                             | boolean            | —                                                                                      | true        |
+| size                                  | size of Input                                            | string             | large / default / small                                                                | default     |
+| placeholder                           | placeholder in non-range mode                            | string             | —                                                                                      | —           |
+| name                                  | same as `name` in native input                           | string             | —                                                                                      | —           |
+| prefix-icon                           | Custom prefix icon component                             | string / Component | —                                                                                      | Clock       |
+| clear-icon                            | Custom clear icon component                              | string / Component | —                                                                                      | CircleClose |
+| start                                 | start time                                               | string             | —                                                                                      | 09:00       |
+| end                                   | end time                                                 | string             | —                                                                                      | 18:00       |
+| step                                  | time step                                                | string             | —                                                                                      | 00:30       |
+| min-time                              | minimum time, any time before this time will be disabled | string             | —                                                                                      | 00:00       |
+| max-time                              | maximum time, any time after this time will be disabled  | string             | —                                                                                      | —           |
+| format <el-tag>1.3.0-beta.3+</el-tag> | set format of time                                       | string             | see [formats](https://day.js.org/docs/en/display/format#list-of-all-available-formats) | HH:mm       |
```

Comment:
remove el-tag, since there's some wired issue related to crowdin 
ref https://github.com/element-plus/element-plus/commit/bb1e86a1f86331000c141c74d8af50eb7813a122

Reply:
ok